package.json

```diff
@@ -77,7 +77,7 @@
     "husky": "^4.3.0",
     "jest": "^26.6.1",
     "netlify-cli": "^3.29.14",
-    "next": "^10.0.8",
+    "next": "^10.2.0",
```

Comment:
This is a breaking change for users of `next` `>=10.0.8 <10.2.0`. 
Should we break builds for those users, or should we support another backport release (like `v1`)?
Note: the first option (breaking builds) might be ok if we think not many users are using those versions (do we have data on this?).